Assets/Editor/AutoBuilder.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.IO;
```

Comment:
Did we ever get this auto builder working?

Reply:
The Auto Builder in  here works in the sense that it does actually make builds - though getting it to work as part of a CI / CD will take more integration work.